src/ducks/Watchlists/Widgets/Table/CompareInfo/Actions/Delete.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState, useCallback, useMemo } from 'react'
+
+import { store } from '../../../../../../redux';
+import { useProjectID, useDeleteWatchlistItems, useAddWatchlistItems } from './hooks';
+import { showNotification } from '../../../../../../actions/rootActions';
+import DarkTooltip from '../../../../../../components/Tooltip/DarkTooltip'
+import NotificationActions from '../../../../../../components/NotificationActions/NotificationActions';
+
+// Styles
+import tableStyles from '../../AssetsTable.module.scss'
+
+const Delete = ({selected, watchlist, refetchAssets}) => {
+    // GraphQL functions
+    const {projectIds} = useProjectID(selected);
+    const {removeWatchlistItems} = useDeleteWatchlistItems();
+    const {addWatchlistItems} = useAddWatchlistItems();
+
+    // UI variables
+    const [isHover, setIsHover] = useState(false);
+    const selectedText = useMemo(() => {
+        return `${selected.length} ${selected.length > 1 ? 'items' : 'item'}`;
+    }, [selected]);
+
+    const deleteClickHandler = useCallback(async() => {
+        try {
```

Comment:
Is it possible to click multiple times on the delete action? If yes, then we should ignore next invocations if there is a loading one until it's finished